test/e2e/network/dns_configmap.go

```diff
@@ -193,3 +193,46 @@ var _ = SIGDescribe("DNS configMap nameserver", func() {
 		t.run()
 	})
 })
+
+type dnsPtrFwdTest struct {
+	dnsTestCommon
+}
+
+func (t *dnsPtrFwdTest) run() {
+	t.init()
+
+	t.createUtilPod()
+	defer t.deleteUtilPod()
+
+	t.createDNSServerWithPtrRecord()
+	defer t.deleteDNSServerPod()
+
+	t.setConfigMap(&v1.ConfigMap{Data: map[string]string{
+		"stubDomains":         fmt.Sprintf(`{"acme.local":["%v"]}`, t.dnsServerPod.Status.PodIP),
+		"upstreamNameservers": fmt.Sprintf(`["%v"]`, t.dnsServerPod.Status.PodIP),
+	}})
+
+	moreForeverTestTimeout := 2 * 60 * time.Second
+
+	t.checkDNSRecordFrom(
+		"123.2.0.192.in-addr.arpa",
+		func(actual []string) bool { return len(actual) == 1 && actual[0] == "my.test." },
+		"dnsmasq",
+		moreForeverTestTimeout)
+
+	t.c.CoreV1().ConfigMaps(t.ns).Delete(t.name, nil)
+	t.checkDNSRecordFrom(
+		"123.2.0.192.in-addr.arpa",
+		func(actual []string) bool { return len(actual) == 0 },
+		"dnsmasq",
+		moreForeverTestTimeout)
+}
+
+var _ = SIGDescribe("DNS configMap nameserver", func() {
```

Comment:
This seems to be in the same category as the other test (both testing `DNS configMap nameserver`), can we combine them?

Reply:
Combined.